coretex/cli/modules/node.py

```diff
@@ -255,8 +255,28 @@ def _configureInitScript() -> str:
     return str(path)
 
 
+def validateConfiguration(config: Dict[str, Any]) -> None:
+    cpuLimit, ramLimit = docker.getResourceLimits()
+
+    if not isinstance(config["nodeRam"], int):
+        raise TypeError(f"Invalid config \"nodeRam\" field type \"{type(config['nodeRam'])}\". Expected: \"int\"")
+
+    if not isinstance(config["cpuCount"], int):
+        raise TypeError(f"Invalid config \"cpuCount\" field type \"{type(config['cpuCount'])}\". Expected: \"int\"")
+
+    if cpuLimit < config["cpuCount"]:
+        raise RuntimeError(f"Configuration not valid. CPU limit in Docker Desktop ({cpuLimit}) is lower than the configured value ({config['cpuCount']})")
+
+    if ramLimit < config["nodeRam"]:
+        raise RuntimeError(f"Configuration not valid. RAM limit in Docker Desktop ({ramLimit}) is lower than the configured value ({config['nodeRam']})")
+
+    if config_defaults.MINIMUM_RAM_MEMORY > config["nodeRam"]:
+        raise RuntimeError(f"Configuration not valid. Minimum Node RAM requirement ({config_defaults.MINIMUM_RAM_MEMORY}) is higher than the configured value ({config['nodeRam']})")
```

Comment:
Do not raise exceptions for these messages, print out all of them at once and return True/False if config is valid or not